spec.md

```diff
@@ -348,16 +348,20 @@ It MUST match the following regular expression:
 
 The `<length>` is the content-length, in bytes, of the current chunk.
 
-Each successful chunk upload MUST have a `202 Accepted` response code, and MUST have the following header:
+Each successful chunk upload MUST have a `202 Accepted` response code, and MUST have the following headers:
 
 ```
-Location <location>
+Location: <location>
+Range: 0-<end-of-range>
```

Comment:
for this to be a new "_MUST_", do we have an idea of whether registries are already doing this?

Reply:
That got pulled in from the Docker Registry spec, so anyone implementing that would already be doing this.